specifyweb/frontend/js_src/lib/components/Attachments/RecordSetAttachment.tsx

```diff
@@ -110,7 +144,17 @@ export function RecordSetAttachments<SCHEMA extends AnySchema>({
       {showAttachments && (
         <Dialog
           buttons={
-            <Button.DialogClose>{commonText.close()}</Button.DialogClose>
+            <>
+              <Button.Info
+                disabled={fetchedCount.current !== records.length || records.length <= 1}
```

Comment:
it may not be clear to uset that all attachments need to be bulked fetched by front-end for them to be downloadable
could you add a `title={}` with explanation to this button when it is disabled?